Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityProfileCloneWindow.cs

```diff
@@ -0,0 +1,320 @@
+                    ProfileCloneBehavior.UseExisting,
+                    subProfileProperty,
+                    subProfileProperty.objectReferenceValue,
+                    subProfileType));
+            }
+
+            Vector2 minWindowSize = MinWindowSizeBasic;
+            minWindowSize.y = Mathf.Max(minWindowSize.y, subProfileActions.Count * SubProfileSizeMultiplier);            
+            cloneWindow.minSize = minWindowSize;
+
+            // If there are no sub profiles, limit the max so the window isn't spawned too large
+            if (subProfileActions.Count <= 0)
+                cloneWindow.maxSize = minWindowSize;
+        }
+
+        private void OnGUI()
+        {
+            if (cloneWindow == null || parentProfile == null || childProfile == null)
+            {
+                Close();
+                return;
+            }
+
+            EditorGUILayout.BeginVertical(EditorStyles.helpBox);
+            EditorGUILayout.ObjectField("Cloning profile", childProfile, typeof(BaseMixedRealityProfile), false);
+            EditorGUILayout.ObjectField("from parent profile", parentProfile, typeof(BaseMixedRealityProfile), false);
+            EditorGUILayout.EndVertical();
+            EditorGUILayout.Space();
+
+            if (subProfileActions.Count > 0)
+            {
+                EditorGUILayout.HelpBox("This profile has sub-profiles. By defult your clone will reference the existing profiles. If you want to specify a different profile, or if you want to clone the sub-profile, use the options below.", MessageType.Info);
+
+                EditorGUILayout.BeginVertical();
+
+                for (int i = 0; i < subProfileActions.Count; i++)
+                {
+                    GUI.color = Color.white;
+                    EditorGUILayout.Space();
+
+                    SubProfileAction action = subProfileActions[i];
+
+                    action.Behavior = (ProfileCloneBehavior)EditorGUILayout.EnumPopup(action.Property.displayName, action.Behavior);
+
+                    switch (action.Behavior)
+                    {
+                        case ProfileCloneBehavior.UseExisting:
+                            GUI.color = Color.Lerp(Color.white, Color.clear, 0.5f);
+                            EditorGUILayout.ObjectField("Existing", action.Property.objectReferenceValue, action.ProfileType, false);
+                            break;
+
+                        case ProfileCloneBehavior.UseSubstitution:
+                            action.SubstitutionReference = EditorGUILayout.ObjectField("Substitution", action.SubstitutionReference, action.ProfileType, false);
+                            break;
+
+                        case ProfileCloneBehavior.CloneExisting:
+                            if (action.Property.objectReferenceValue == null)
+                            {
+                                EditorGUILayout.LabelField("Can't clone profile - none is set.");
```

Comment:
nit: change to "Can't clone profile - no profile is currently set." ? It could be more clear how to fix things.